crud/crud.cpp

```diff
@@ -128,11 +127,13 @@ crud::handle_create(const database_msg& request, std::shared_ptr<bzn::session_ba
 
 
 void
-crud::handle_read(const database_msg& request, std::shared_ptr<bzn::session_base> session)
+crud::handle_read(const bzn::caller_id_t& /*caller_id*/, const database_msg& request, std::shared_ptr<bzn::session_base> session)
 {
```

Comment:
I take it that we are not doing read permissions, at this time... So anybody can read.

Reply:
yes for mvp